docs/source/cookbook/optical_mosaic.rst

```diff
@@ -147,106 +147,106 @@ Date
+The number of images in one single season of one year may not be enough to produce a correct mosaic. SEPAL provides two secondary sliders to increase the pool of images to create the mosaic. Both count the number of seasons SEPAL can retrieve in the past (:code:`Past season` - (3)) and in the future (:code:`Future season` - (4)). 
 
-When the selection is done click on :icon:`fa fa-check` :guilabel:`apply` button.
+When the selection is done click on the :icon:`fa fa-check` :guilabel:`Apply` button.
 
 .. thumbnail:: ../_images/cookbook/optical_mosaic/select_season.png
-    :title: The season selection panel
+    :title: The season selection panel.
     :group: optical-mosaic-recipe
 
 Sources
 ^^^^^^^
 
-As mentioned in the introduction, a mosaic uses different raster datasets and they can be obtained from multiple sources. SEPAL allows you to select data from multiple entry points, below you can find a description of these sources (click on the link to see the corresponding dataset information):
+As mentioned in the introduction, a mosaic uses different raster datasets that can be obtained from multiple sources. SEPAL allows you to select data from multiple entry points. Below, you can find a description of these sources (click on the link to see the corresponding dataset information):
 
--   :guilabel:`L8`: `Landsat 8 Tier 1 <https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C01_T1>`__. Landsat scenes with the highest available data quality are placed into Tier 1 and are considered suitable for time-series processing analysis. Tier 1 includes Level-1 Precision Terrain (L1TP) processed data that have well-characterized radiometry and are inter-calibrated across the different Landsat sensors. The geo-registration of Tier 1 scenes will be consistent and within prescribed tolerances [<=12 m root mean square error (RMSE)]. All Tier 1 Landsat data can be considered consistent and inter-calibrated (regardless of the sensor) across the full collection.
+-   :guilabel:`L8`: `Landsat 8 Tier 1 <https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C01_T1>`__. Landsat scenes with the highest available data quality are placed into Tier 1 and considered suitable for time-series processing analysis. Tier 1 includes Level-1 Precision Terrain (L1TP) processed data that have well-characterized radiometry and are inter-calibrated across the different Landsat sensors. The geo-registration of Tier 1 scenes will be consistent and within prescribed tolerances (<=12 m root mean square error [RMSE]). All Tier 1 Landsat data can be considered consistent and inter-calibrated (regardless of the sensor used) across the full collection.
```

Comment:
this text is actually copy/pasted from the actual satellite documentation, even though it's not perfect, it should not be changed. after links that starts with "developers.google.com, I usually simply copy/pastde the doc, you can discard these lines